src/promptflow/promptflow/_cli/_pf_azure/_flow.py

```diff
@@ -63,15 +77,41 @@ def add_parser_flow_create(subparsers):
 
 def add_parser_flow_list(subparsers):
     """Add flow list parser to the pf flow subparsers."""
-    add_params = [_set_workspace_argument_for_subparsers] + logging_params
+    epilog = """
+Examples:
+
+# List flows:
+pfazure flow list
+# List most recent 10 runs status:
+pfazure flow list --max-results 10
+# List flows that are owned by current user:
+pfazure flow list --owned-only
```

Comment:
Not sure if we should set `owned-only` default to be True, in that case we need to rename this parameter to something like `--include-others`, @wangchao1230 do you have any comments?